tests/ci_build/conda_env/macos_cpu_test.yml

```diff
@@ -0,0 +1,35 @@
+name: cpu_test
```

Comment:
How's this different from `cpu_test.yml`

Reply:
This one is missing numba and llvmlite.